README.md

```diff
@@ -98,6 +102,7 @@ Here are the benchmark rendering results to compare visual results:
 * [Manual layout rendering result](docs_markdown/benchmark_result_ManualLayout.png)
 * [NKFrameLayoutKit rendering result](docs_markdown/benchmark_result_NKFrameLayoutKit.png)
 * [PinLayout rendering result](docs_markdown/benchmark_result_PinLayout.png)
+* [Texture rendering result](docs_markdown/benchmark_result_Texture.png)
```

Comment:
In the rendering results, there are some issues:

1. The top section should look like this 
<img width="481" alt="image" src="https://user-images.githubusercontent.com/14981341/92036453-3d466400-ed3e-11ea-9d3d-dc7561fc31ab.png">
- Wrong vertical alignment
- Image size
- Orange background color

2. Missing some bottom padding. It should look like this
<img width="523" alt="image" src="https://user-images.githubusercontent.com/14981341/92036605-7ed70f00-ed3e-11ea-89d6-522e2adc6fed.png">

3. Missing some bottom padding. It should look like this
<img width="522" alt="image" src="https://user-images.githubusercontent.com/14981341/92036654-944c3900-ed3e-11ea-9e07-5ba6a106f593.png">

4. The label should be vertically aligned. It should look like this
<img width="498" alt="image" src="https://user-images.githubusercontent.com/14981341/92036746-b34acb00-ed3e-11ea-8c1b-034194768696.png">

Reply:
Rendering results issues should be fixed now @lucdion .